src/components/TodoList/TodoList.tsx

```
@@ -1,100 +1,87 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+interface TodoListProps {
+  todos: Todo[];
+  onClickSetUserId: (v: number) => void;
+  onClickSetTodo: (v: Todo) => void;
+  oneTodoForCheck: Todo | null;
+}
 
-    <tbody>
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">1</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">delectus aut autem</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-      <tr data-cy="todo" className="has-background-info-light">
-        <td className="is-vcentered">2</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">quis ut nam facilis et officia qui</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye-slash" />
-            </span>
-          </button>
-        </td>
-      </tr>
+export const TodoList: React.FC<TodoListProps> = ({
+  todos,
+  onClickSetUserId = () => {},
+  onClickSetTodo = () => {},
```

Issue: You are providing default values for `onClickSetUserId` and `onClickSetTodo` in the destructuring assignment. Since these props are required in the `TodoListProps` interface and always provided by the parent, default values are unnecessary and could hide missing prop issues. Remove the default values to ensure prop requirements are enforced.